interfaces/PA-App/src/app/personal/personal.page.html

```diff
@@ -1,7 +1,7 @@
 <ion-header>
   <ion-toolbar color="dark">
     <ion-title>
-      {{ 'tabs.you'|translate }}
+      {{ mode === 'batch' ? paBatch.length + ' PAs queued' : 'tabs.you'|translate }}
```

Comment:
I think it is cleaner to add a separate, optional 'element' to indicate the status of this feature.

So:

```suggestion
      {{ 'tabs.you'|translate }}
      <span *ngIf="mode === 'batch'" class="ion-margin-start">
      {{ 'mode.batch.title'|translate:{ number: paBatch.length }}
      </span>
```
And the translations-file would include:
```json
"mode": {
  "batch": {
    "title": "{{number}} PAs queued"
  }
}
```